backend/dissemination/api/api_v1_0_3/create_access_tables.sql

```diff
@@ -0,0 +1,22 @@
+-- This is explicitly not a Django managed table.
+-- In order to grant an API key access to read tribal data
+-- we need to go through a PR.
```

Comment:
This comment isn't quite correct, right? An admin user can also grant access via API?

Reply:
No. The only way to get access tot he API is to have your key's UUID added to the access table. The text says "grant an API key access," not "grant a user access."

To gain access as a user, we use the API. To use the API, we commit to this table.